### PR TITLE
SOFT4 end of tournament

### PR DESCRIPTION
**Pull request by WalterToro**
whoa, late af

---

### CODE REVIEW

File: wiki/Tournaments/SOFT/4/en.md [hunk 1]
```diff
@@ -27,11 +27,17 @@ The **Springtime Osu!mania Free-for-all Tournament 4** (***SOFT 4***), is a one-
...
 | Finals | 2019-06-08/2019-06-09 |
-| Grand Finals | 2019-06-22/2019-06-23 |
+| Grand Finals | 2019-06-22 |
 
 ## Prizes
 
-*Prizes for this tournament are to be decided.*
+| Placing | Prize |
+| :-: | :--- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 50 USD and a profile badge |
```
Comment (cl8n) [inline wiki/Tournaments/SOFT/4/en.md]:
```suggestion
| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 50 USD, profile badge |
```
think this is more standard with the other tournament pages

File: wiki/Tournaments/SOFT/4/en.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ tags:
 
 > SOFT 4 When?, now!
```
Comment (cl8n) [inline wiki/Tournaments/SOFT/4/en.md]:
```suggestion
> SOFT 4 when?

Now!
```
"now!" isn't part of the quote

File: wiki/Tournaments/SOFT/4/en.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ tags:
...
 
-The **Springtime Osu!mania Free-for-all Tournament 4** (***SOFT 4***), is a one-on-one, country-independent osu!mania 4-key tournament.
+The **Springtime Osu!mania Free-for-all Tournament 4** (***SOFT 4***), was a one-on-one, country-independent osu!mania 4-key tournament.
```
Comment (TPGPL) [inline wiki/Tournaments/SOFT/4/en.md]:
```suggestion
The **Springtime Osu!mania Free-for-all Tournament 4** (***SOFT 4***) was a one-on-one, country-independent osu!mania 4-key tournament.
```

File: wiki/Tournaments/SOFT/4/en.md [hunk 4]
```diff
@@ -27,11 +27,17 @@ The **Springtime Osu!mania Free-for-all Tournament 4** (***SOFT 4***), is a one-
 | Quarterfinals | 2019-05-24/2019-05-26 |
 | Semifinals | 2019-06-01/2019-06-02 |
 | Finals | 2019-06-08/2019-06-09 |
-| Grand Finals | 2019-06-22/2019-06-23 |
+| Grand Finals | 2019-06-22 |
 
 ## Prizes
 
-*Prizes for this tournament are to be decided.*
+| Placing | Prize |
```
Comment (TPGPL) [inline wiki/Tournaments/SOFT/4/en.md]:
```suggestion
| Placing | Prizes |
```

**Pull request merged**